### ISSUE TITLE
"Execute SQL Statement" acts like "Execute SQL in new tab"

### ISSUE THREAD

**Issue by baztastic** [dbeaver/dbeaver#10425]
#### System information: 
- MacOS Catalina
- DBeaver Version 7.2.4.202011011603
- Java version 
  - openjdk 15.0.1 2020-10-20
  - OpenJDK Runtime Environment (build 15.0.1+9-18)
  - OpenJDK 64-Bit Server VM (build 15.0.1+9-18, mixed mode, sharing)

#### Connection specification:
- Database `snowflake`
- Driver name JDBC
- HTTPS & SOCKS proxies in use

#### Describe the problem you're observing:
When I open a sql script, initially the "Execute SQL Statement" button or hotkey acts correctly and runs each statement in the same tab, however after the first time I use "Execute SQL in new tab", the previous button always creates a new tab for each statement. The same thing happens with the "Execute SQL Script" where it acts like "Execute Statements in Multiple Tabs".

If I right-click on the results pane and select "Close multiple results" the behaviour goes back to normal until I click "Execute SQL in new tab" again.

**Comment by kseniiaguzeeva** [comment]
Thank you for the report. Should be fix as part of https://github.com/dbeaver/dbeaver/issues/10280. You can test EA version https://dbeaver.io/files/ea/

**Comment by baztastic** [comment]
Thanks for the update @kseniiaguzeeva - I tried the EA version and still have the same issue - here's a quick screen recording:

![img](https://media.giphy.com/media/EezQJA2v7T1e7UE0Ac/giphy.gif)

It happened after I updated so maybe something got screwed up along the way. What's the best way to nuke my installation and start afresh?

**Comment by kseniiaguzeeva** [comment]
Just close DBeaver, download  and install a new version.

**Comment by kseniiaguzeeva** [comment]
Managed to reproduce another wrong behaviour.
Steps:
1. write the script:
сделать execute SQL Script
закрыть 2 и 3 вкладку резалтсета и сделать execute SQL statement для второго стэйтмента 2 раза
сделать execute SQL statement для третьего стейтмента
сделать  execute SQL statement in new tab  для третьего стейтмента
сделать execute SQL statement  для третьего стейтмента
Ожидается: существующие вкладки закрываются, появляется одна из последнего запроса

**Comment by kseniiaguzeeva** [comment]
My last comment is the expected behaviour from our last fix since 7.3.1 (already available in EA version). I still cannot reproduce your result from gif.
Could you please try the latest EA (or wait for release 7.3.1) https://dbeaver.io/files/ea with my steps?
1 write the script for Snowflake:
```
create OR REPLACE temporary table mytemptable (
id number, creation_date date
);
SELECT * FROM mytemptable;
SELECT id FROM mytemptable;
```
2 Press Execute SQL Script -3 tabs appear
3 Close 2-3 tabs in the ResultSet, place the cursor on the 3d statement and press Execute SQL statement - the result of the 3rd statement appears in the  tab
4 Press Execute SQL statement in new tab for the 3d statement - 2 tabs are in ResultSet. Both are with the result of 3d query.
5 Place the cursor on the 2d statement and press execute SQL statement - the content of the second tab was changed (the result of the 2nd query is in it). The first tab remains in the ResultSet.

Here is wiki article about execution behaviour https://github.com/dbeaver/dbeaver/wiki/SQL-Execution#result-tabs

**Comment by uslss** [comment]
there is no update on the issue for a long time.
if it is still actual for you - feel free to reopen it with an additional info
